prow/config.yaml

```diff
@@ -400,6 +400,30 @@ tide:
     - needs-rebase
   - repos:
     - kubernetes/kubernetes
+    milestone: v1.13
+    includedBranches:
+    - master
+    - release-1.13
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/blocked-paths
+    - do-not-merge/cherry-pick-not-approved
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/release-note-label-needed
+    - do-not-merge/work-in-progress
+    - needs-kind
+    - needs-rebase
+    - needs-sig
```

Comment:
This also needs to include `needs-priority`

Reply:
Done.